src/lib/containers/UserCardListRotate.tsx

```diff
@@ -16,32 +16,32 @@ export type UserCardListRotateProps = {
   token?: string
   useQueryResultUserData?: boolean
   size?: UserCardSize
-  summaryLink: string
-  summaryLinkText: string
+  summaryLink?: string
```

Comment:
Made summary link/text optional.  If not included, then button will not be displayed.